File: resoluciones/python/tareas5.py
#1))

numeros = [6,5,4,3,2,1]

def ordenarArr(arr):
    for i in range(0,len(arr),1):
        ayudante = 0
        for j in range(i + 1,len(arr),1):
            if(arr[i] > arr[j]):
                ayudante = arr[i]
                arr[i] = arr[j]
                arr[j] = ayudante
    return arr

print(ordenarArr(numeros))

#2)
import random

numeros2 = [6,5,4,3,2,1]
numeroObjetivo = [numeros[random.randint(0,5)]]

def numerObjetivo(arr, numObj):
    for i in range(0,len(arr),1):
        for j in range(i + 1,len(arr),1):
            if(arr[i] + arr[j] == numObj):
                print(f"el numero {arr[i]} + el numero {arr[j]} es igual a: {numObj}")

print(numerObjetivo(numeros))

File: resoluciones/javascript/tareas5.js
//1)
let numeros = [3,8,15,-2,1]

function ordenarValores(arr) {
    for (let i = 0 ; i < arr.length; i++) {
       let CambioDeNumeros;
        for(let j = i + 1; j < arr.length; j++) {
            if (arr[j] < arr[i]) {
                CambioDeNumeros = arr[i];
                arr[i] = arr[j];
                arr[j] = CambioDeNumeros;
            };
        };
    };
}

console.log(ordenarValores(numeros))

//2)
let numerosDados = [8,5,18,4]
let numeroObjetivo = 13

function seleccionar(arr,No) {
    for (let i = 0 ; i < arr.length; i++) {
        for(let j = i + 1; j < arr.length; j++) {
            if (arr[j] + arr[i] == No) {
                return arr[j] + " y " + arr[i] + " son igual que " + No
            };
        };
    };
}

console.log(seleccionar(numerosDados,numeroObjetivo))
